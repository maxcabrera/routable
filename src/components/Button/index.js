import styled from 'styled-components'

const Button = styled.button`
  display:block;
  border: none;
  box-sizing: border-box;
  flex-grow: 1;
  padding: 10px;
  background-color: #3056ba;
  color: #fff;
  height: 56px;
  line-height: 28px;
  font-weight: 600;
  font-size: 14px;
  width: 100%;
  margin-top: 14px;
  border-radius: 8px;    
  box-shadow: 0 2px 6px rgba(77,92,113,.3); 
  
  &:hover {
    cursor: pointer;
    box-shadow: 0 6px 6px rgba(77,92,113,.3);
  }
`

export default Button