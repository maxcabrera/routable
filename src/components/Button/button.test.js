import Button from './'

describe('Button Component', () => {
  test('Snapshot test for Header', () => {
    const tree = renderer.create(<Button>Click me</Button>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})