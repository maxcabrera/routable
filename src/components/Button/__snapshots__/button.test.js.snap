// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Snapshot test for Header 1`] = `
<button
  className="sc-bdVaJa cOBoFD"
>
  Click me
</button>
`;
